test/index.test.js

```diff
@@ -27,6 +30,35 @@ describe('plugin', () => {
       await app.receive(event)
       expect(sync).toHaveBeenCalled()
     })
+
+    it('creates a check', async () => {
+      await app.receive(event)
+      expect(github.checks.create).toHaveBeenCalled()
+    })
```

Comment:
this test seems redundant based on the ones below. if so, could you remove this one?

Reply:
it did become redundant after i created the more specific test for success and failure 
https://github.com/probot/settings/pull/149/commits/2599ea47fe32a828dca1fc808cf319b4a4f7212b